### ISSUE TITLE
Update docs:  inputToken --> token

### ISSUE THREAD

**Issue by gavares** [winstonjs/winston-loggly#14]
`git clone` from master and then run `npm test` gives error: 

```
/tmp/winston-loggly/lib/winston-loggly.js:36
    throw new Error('Loggly Customer token is required.');
```

Changing config option `inputToken: myToken` to `token: myToken` and the tests pass.

**Comment by adamhadani** [comment]
:+1: 
I could reproduce, and changing as mentioned above fixed it for me.
One test still fails though which I suspect is unrelated:

```
  An instance of the Loggly Transport when passed an input token the log() method when passed circular metadata
    ✗ should log messages to loggly
        » expected null, got undefined // winston-loggly-test.js:46
```

**Comment by indexzero** [comment]
Could you rephrase your comment in the form of a pull-request?

**Comment by adamhadani** [comment]
@indexzero  opened PR #32
